src/Release/CreateRelease.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Clearbooks\Labs\Release;
+use Clearbooks\Labs\Release\Gateway\ReleaseGateway;
+use Clearbooks\Labs\Release\UseCase\CreateRelease\Request;
+
+/**
+ * @author: Ryan Wood <ryanw@clearbooks.co.uk>
+ * @created: 15/07/15
+ */
+class CreateRelease
+{
+    /**
+     * @var ReleaseGateway
+     */
+    private $releaseGateway;
+
+    /**
+     * @param ReleaseGateway $releaseGateway
+     */
+    function __construct( ReleaseGateway $releaseGateway )
+    {
+        $this->releaseGateway = $releaseGateway;
+    }
+
+
+    /**
+     * @param Request $request
+     * @return Release|bool
+     */
+    public function execute( Request $request )
+    {
+        if( !$this->isValidReleaseRequest( $request ) ) {
+            return false;
+        }
+
+        return $this->releaseGateway->addRelease( $request->getReleaseName(), $request->getReleaseInfoUrl() );
```

Comment:
I'd avoid returning a data structure used by gateway from your interactor as it'll couple whatever calls it to the specific implementation of this interactor that happens to use this gateway

Maybe some kind of release ID?